src/forms/FormActions.js

```diff
@@ -87,207 +94,129 @@ export const GALLERY_ENABLE_IDENTIFIABLE = 'GALLERY_ENABLE_IDENTIFIABLE';
-  }
+/**
+ * Action creators
+ */
 
-  return init;
-};
+const formRequestStarted = id => ({ type: FETCH_FORM_REQUEST, id });
+const formRequestSuccess = form => ({ type: FETCH_FORM_SUCCESS, form });
+const formRequestFailure = err => ({ type: FETCH_FORM_FAILURE, err });
 
-export const formRequestStarted = id => {
-  return {
-    type: FORM_REQUEST_STARTED,
-    id
-  };
-};
+const formsRequestStarted = () => ({ type: FETCH_FORMS_REQUEST });
+const formsRequestSuccess = forms => ({ type: FETCH_FORMS_SUCCESS, forms });
+const formsRequestFailure = err => ({ type: FETCH_FORMS_SUCCESS, err });
 
-export const formRequestSuccess = form => {
-  return {
-    type: FORM_REQUEST_SUCCESS,
-    form
-  };
-};
+const deleteSuccessful = id => ({ type: FORM_DELETED, id });
 
-export const formRequestFailure = (err) => {
-  return {
-    type: FORM_REQUEST_FAILURE,
-    err
-  };
-};
+const formRequestEditAccess = formId => ({
+  type: REQUEST_EDIT_FORM_ACCESS,
+  formId,
+  publish: true
+});
 
-export const formsRequestStarted = () => {
-  return {
-    type: FORMS_REQUEST_STARTED
-  };
-};
+const formLeaveEdit = formId => ({ type: EDIT_FORM_LEAVE, formId, publish: true });
 
-export const formsRequestSuccess = forms => {
-  return {
-    type: FORMS_REQUEST_SUCCESS,
-    forms
-  };
-};
+const createEmptyAction = saveDestination => ({
+  type: CREATE_EMPTY_FORM,
+  saveDestination
+});
 
-export const formsRequestFailure = err => {
-  return {
-    type: FORMS_REQUEST_FAILURE,
-    err
-  };
-};
+const submissionsFetched = (counts, submissions) => ({
+  type: FETCH_SUBMISSIONS_SUCCESS,
+  submissions,
+  counts
+});
 
+const submissionsFetchError = error => ({ type: FETCH_SUBMISSIONS_FAILED, error });
 
-export const deleteSuccessful = (id) => {
-  return {
-    type: FORM_DELETED,
-    id
-  };
-};
+const formCreated = form => ({ type: FORM_CREATED, form });
+const formCreationFailure = error => ({ type: FORM_CREATION_FAILURE, error });
 
-export const formRequestEditAccess = formId => {
-  return {
-    type: FORM_REQUEST_EDIT_ACCESS,
-    formId,
-    publish: true
-  };
-};
+const requestGallery = () => ({ type: FETCH_FORM_GALLERY_REQUEST });
+const receivedGallery = gallery => ({ type: FETCH_FORM_GALLERY_SUCCESS, gallery });
+const galleryRequestError = error => ({ type: FETCH_FORM_GALLERY_FAILED, error });
 
-export const formLeaveEdit = formId => {
-  return {
-    type: FORM_EDIT_LEAVE,
-    formId,
-    publish: true
-  };
-};
+const answerRemovedFromGallery = gallery => ({
+  type: FORM_ANSWER_REMOVED_FROM_GALLERY,
+  gallery
+});
 
-export const fetchForms = () => {
-  return (dispatch, getState) => {
-    dispatch(formsRequestStarted());
+/**
+ * Actions API
+ */
 
-    fetch(`${getState().app.pillarHost}/api/forms`)
-      .then(res => res.json())
-      .then(forms => dispatch(formsRequestSuccess(forms)))
-      .catch(error => dispatch(formsRequestFailure(error)));
-  };
+export const fetchForms = () => (dispatch, getState) => {
```

Comment:
This feels weird. I'm not sure about having the beginning of the return statement on the same line as the function definition. Have you seen this in other places?